egg/core/interaction.py

```diff
@@ -20,16 +20,17 @@ class LoggingStrategy:
     store_message_length: bool = True
 
     def filtered_interaction(
-        self,
-        sender_input: Optional[torch.Tensor],
-        receiver_input: Optional[torch.Tensor],
-        labels: Optional[torch.Tensor],
-        message: Optional[torch.Tensor],
-        receiver_output: Optional[torch.Tensor],
-        message_length: Optional[torch.Tensor],
-        aux: Dict[str, torch.Tensor],
-    ):
+            self,
+            sender_input: Optional[torch.Tensor],
+            receiver_input: Optional[torch.Tensor],
+            labels: Optional[torch.Tensor],
+            message: Optional[torch.Tensor],
+            receiver_output: Optional[torch.Tensor],
+            message_length: Optional[torch.Tensor],
+            aux: Dict[str, torch.Tensor],
+            aux_input: Dict[str, Any] = None,
```

Comment:
I think having it as optional in the typing definitin (as long as it's before aux I guess) will set it to None by default

Reply:
I don't remember where but it raises an error if I use:
```
 aux_input: Optional[Dict[str, Any]]
```

Reply:
mmh, any ideas on what that was?